client/js/components/support/DpSupportCard.vue

```diff
@@ -8,14 +8,14 @@ All rights reserved
 </license>
 
 <template>
-  <section class="bg-color--white">
```

Comment:
I think with this changes, You break the page `/informationen`

Reply:
that has no impact on the support card because the outer <li> tag has the same class. However, you're right, if we want to reuse the component in the future, the styling should be in the component.. 

adjust it here https://github.com/demos-europe/demosplan-core/pull/2300/commits/4fec05388b0f54bca521f3ad8dbeecdaf5aaf03d

Reply:
do not understand, why `$dp-color-blue-light-3;` is a background-color of the c-support__wrapper class? The card is white (error page - gray), and we don't use it anywhere that requires a blue color.. I changed it to white, so it fix the problem.. if it is ok for you?